benchmarks/bench_logging_overhead.py

```
@@ -0,0 +1,350 @@
+"""
+
+import os
+import sys
+import time
+import torch
+import numpy as np
+from typing import List, Tuple
+
+# Get logging level BEFORE importing flashinfer
+LOGGING_LEVEL = int(os.environ.get("FLASHINFER_LOGLEVEL_DBG", "0"))
+LOG_DEST = os.environ.get("FLASHINFER_LOGDEST_DBG", "/tmp/flashinfer_benchmark_log.txt")
```

Ruff (0.14.5)

Probable insecure usage of temporary file or directory: "/tmp/flashinfer_benchmark_log.txt"

(S108)